log/sequencer.go

```diff
@@ -24,6 +23,10 @@ type Sequencer struct {
 	keyManager crypto.KeyManager
 }
 
+// CurrentRootExpiredFunc examines a signed log root and decides if it has expired with respect
```

Comment:
Well if we're into the realms of imagination ...

I think this is a good idea but I'd like to defer until I've got a 1st cut at serving proofs done. Otherwise we were going to have to have public / private tree heads to cope with cases where you're not allowed to issue STHs too frequently.